### PR TITLE
Create-a-plugin-intro-and-set-up

### PR DESCRIPTION
**Pull request by Pierregvx**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]

**Comment by Thykof** [comment]
I you don't ask to install goswagger, it will fail at the next part when running go generate

**Comment by Pierregvx** [comment]
> I you don't ask to install goswagger, it will fail at the next part when running go generate

thanks for the catch ! I'll add it

File: docs/tutorial/massa-station-plugin/set-up-plugin.mdx [hunk 1]
```diff
@@ -1,45 +1,56 @@
...
+:::
 
-- Create and navigate to the folder my-plugin
-- Run:
+## Initialize the Go plugin
 
-  ```bash
-  go mod init my-plugin
-  ```
+First, create and navigate to the folder `my-plugin` and then start your module using:
```
Comment (Thykof) [inline docs/tutorial/massa-station-plugin/set-up-plugin.mdx:26]:
```suggestion
First, create and navigate to the folder `my-plugin` and then initialize your project using:
```

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by Thykof — APPROVED** [review]
my suggestion above is no blocking

**Pull request closed without merging**